vector/vector_test.go

```diff
@@ -41,3 +49,88 @@ func FuzzQuery(f *testing.F) {
 		fuzz.CompareValues(t, resultZNG, resultVNG)
 	})
 }
+
+const N = 10000000
+
+func BenchmarkReadZng(b *testing.B) {
```

Comment:
This doesn't really have anything to do with using vectors in the lake so if you want to add it let's do so in a separate PR.